WonSeungHyun/app.js

```diff
@@ -0,0 +1,39 @@
+const http = require("http");
+const express = require("express");
+const cors = require("cors");
+const morgan = require("morgan");
+const dotenv = require("dotenv");
+dotenv.config();
+
+const { DataSource } = require("typeorm");
+const myDataSource = new DataSource({
+  type: process.env.TYPEORM_CONNECTION,
+  host: process.env.TYPEORM_HOST,
+  port: process.env.TYPEORM_PORT,
+  username: process.env.TYPEORM_USERNAME,
+  password: process.env.TYPEORM_PASSWORD,
+  database: process.env.TYPEORM_DATABASE,
+});
+
+myDataSource.initialize().then(() => {
+  console.log("Data Source has been initialized!");
+});
```

Comment:
- myDataSource 보다는 `appDataSource` , `mysqlDataSource` 혹은 간결하게 `database`로 변수명을 변경해주세요. “my”라는 용어는 예제 코드에서는 사용할 수 있으나 실제 코드에서는 피해주세요.
- 추가로 `DataSource.initialize()` 함수가 실행되고 나면, then()과 catch()를 사용해서 에러를 핸들링해줄 수 있습니다. 이렇게 하면, initialize() 함수가 실행되어서 database와 실제로 connection을 생성할 때, 성공했는지 실패했는지 여부를 확인할 수 있습니다.
아래는 예시입니다.
```suggestion
const appDataSource = new DataSource({
  type: process.env.TYPEORM_CONNECTION,
  host: process.env.TYPEORM_HOST,
  port: process.env.TYPEORM_PORT,
  username: process.env.TYPEORM_USERNAME,
  password: process.env.TYPEORM_PASSWORD,
  database: process.env.TYPEORM_DATABASE,
});

appDataSource.initialize().then(() => {
  console.log("Data Source has been initialized!");
});
```